### ISSUE TITLE
M'overlay won't play music on certain stages in Beyond melee

### ISSUE THREAD

**Issue by Bromomento2427** [bkacjios/m-overlay#135]
**Describe the bug**
M'Overlay won't play music on the new scenarios

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Dolphin Version: Slippi 2.3.3

**To Reproduce**
1. Open Beyond Melee
2. Click on any stage that isn't from vanilla Melee 
3. Music won't play despite there being an audio file on the configuration folder for the selected stage

**Expected behavior**
M'Overlay should play music on any stage that has audio files on the configuration folder but that isn't happening on the stages added to Beyond Melee

**Additional context**
For example, m'overlay plays music on Pokemon Stadium and Yoshi's story but it doesn't play music on Galacta battleground, midnight battlefield, etc. I asked this issue on the Beyond Melee discord and a mod responded this: 
![image](https://user-images.githubusercontent.com/99105599/152665528-4c1d225f-8fd7-4605-b3b2-1f9ac456f1f1.png)
![image](https://user-images.githubusercontent.com/99105599/152665536-9f18fa05-5734-4e8f-9c59-1111262f41e7.png)

**Comment by bkacjios** [comment]
Yeah, this has been a problem I'm not really sure how to solve. I would need a way to detect that the rom is any version of Beyond Melee.
